src/index.js

```diff
@@ -0,0 +1,39 @@
+import { MongoClient } from 'mongodb';
+
+const uri = 'mongodb://localhost:27017';
+const client = new MongoClient(uri, { useNewUrlParser: true, useUnifiedTopology: true });
+
+async function getDB() {
+  try {
+    await client.connect();
+    return client.db('kaende');
+  } catch(ex) {
+    console.error(ex)
+  }
+}
+
+
+export const start = async () => {
+    console.log('Connecting to DB...');
+    const db = await getDB();
+
+    const runNotificationCycle = async () => {
+        const invoices = await db.collection('invoices').find({}).toArray();
```

Comment:
Use Mongo DB's filter method to only retrieve invoices which are due today instead of fetching all invoices